Gems/MiniAudio/Code/Include/MiniAudio/MiniAudioListenerBus.h

```diff
@@ -18,8 +18,21 @@ namespace MiniAudio
     public:
         ~MiniAudioListenerRequests() override = default;
 
-        virtual void SetFollowEntity(AZ::EntityId followEntity) = 0;
+        virtual void SetFollowEntity(const AZ::EntityId followEntity) = 0;
         virtual void SetPosition(const AZ::Vector3& position) = 0;
+        virtual AZ::u32 GetChannelCount() const = 0;
+
+        //! Cone controls for directional attenuation
+        virtual float GetInnerAngleInRadians() const = 0;
+        virtual void SetInnerAngleInRadians(const float& innerAngleInRadians) = 0;
+        virtual float GetInnerAngleInDegrees() const = 0;
+        virtual void SetInnerAngleInDegrees(const float& innerAngleInDegrees) = 0;
+        virtual float GetOuterAngleInRadians() const = 0;
+        virtual void SetOuterAngleInRadians(const float& outerAngleInRadians) = 0;
+        virtual float GetOuterAngleInDegrees() const = 0;
+        virtual void SetOuterAngleInDegrees(const float& outerAngleInDegrees) = 0;
+        virtual float GetOuterVolume() const = 0;
+        virtual void SetOuterVolume(const float& outerVolume) = 0;
```

Comment:
I'm looking for it in our coding standards but not finding it - we generally pass simple built in types by value and not by const reference.  This is generally because it is cheaper to copy simple types than dereference.

Reply:
@AMZN-alexpete Let me know if this is part of the standard and I'll be sure to change it to pass by value.

Reply:
@AMZN-alexpete Also, once you do find it, let me know if it applies to `AZ::Vector3` in the case of `MiniAudioPlaybackComponentController::SetDirection`.

Reply:
@AMZN-alexpete I just changed it to to have ordinal types passed by value instead of const reference: 490edf43e0c4a2e0483c119da75228186801000a

Reply:
I left the `AZ::Vector3` passed by const reference though, hopefully that's right.